eWallet/user/templates/register.html

```diff
@@ -0,0 +1,72 @@
+{% extends "base.html" %}
+
+{% block title %}Register Page{% endblock %}
+
+{% block content %}
+
+  <div class="container mt-5">
+    
+    <div class="row justify-content-center">
+      {% comment %} {% if messages %}
+                 
+      {% for message in messages %}
+          {% if message.tags == message.success %}
+              <div class="alert alert-success">
+          {% elif error_message %}
+              <div class="alert alert-danger">
+          {% endif %}
+          <li>{{ error_message }}</li>
+                </div>
+            {% endfor %}
```

Comment:
Don't push commented code. Also make sure these alerts are on app pages. I would place this code in a separate html file and include that file in base.html, and check if there are any alerts then display them.